apps/chat/src/utils/app/common.ts

```diff
@@ -41,6 +41,17 @@ export const isEntityNameOnSameLevelUnique = (
   return !sameLevelEntities.some((e) => nameToBeUnique === e.name);
 };
 
+export const isImportEntityNameOnSameLevelUnique = (
```

Comment:
we already have `isEntityNameOnSameLevelUnique`

Reply:
This is another method. `isEntityNameOnSameLevelUnique` not working in this case.